lib/polyfill/patchedmediakeys_apple.js

```diff
@@ -27,14 +27,29 @@ goog.require('shaka.util.StringUtils');
 shaka.polyfill.PatchedMediaKeysApple = class {
   /**
    * Installs the polyfill if needed.
+   * @param {boolean} enableUninstall enables uninstalling the polyfill
    * @export
    */
-  static install() {
+  static install(enableUninstall = false) {
     if (!window.HTMLVideoElement || !window.WebKitMediaKeys) {
       // No HTML5 video or no prefixed EME.
       return;
     }
 
+    if (enableUninstall) {
+      this.enableUninstall = true;
+      // eslint-disable-next-line no-restricted-syntax
+      this.originalHTMLMediaElementPrototypeMediaKeys = HTMLMediaElement
```

Comment:
This does not work (cannot call get directly on prototype.mediaKeys), and I'm unsure if there's a need for saving and restoring the mediaKeys. Any input would be appreciatedb

Reply:
If you need to backup and restore a getter, you may need to try getOwnPropertyDescriptor.  There is a good example of backing up and restoring a getter this way in test/player_unit.js.

Does this help?